### ISSUE TITLE
Access to YouTubePlayer from other controls when using JetPack Compose

### ISSUE THREAD

**Issue by wthsts** [PierfrancescoSoffritti/android-youtube-player#925]
I'm a newbie to Android Studio, Kotlin and JetPack Compose.  I have been able to successfully get the android-youtube-player to work in my simple app.

Is there a recommended approach to access a YouTubePlayer instance from other controls.  A simple use case where a button onclick event handler needs to call YouTubePlayer function such as loadVideo().  Most of the code I have seen using this library makes use of the findViewById approach to provide and/or propagate access to a YouTubPlayer instance retrieved from AbstractYouTubePlayerListener onReady() function.   From what I understand, findViewById is not part of the JetPack Compose design.

**Comment by PierfrancescoSoffritti** [comment]
Hi, i suggest storing the instance of `YouTubePlayer` in a global variable :)

**Comment by wthsts** [comment]
I thought about that, but didn't want to say it out loud.  Haha.

Thanks for helping me.  I'm in prototype mode and your recommendation unblocks my work.
